.changeset/tender-avocados-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+"gha-workflow-validator": patch
+---
+
+fix: do not validate commented lines
```

Comment:
hmm I wonder which changeset file takes precedence :)

Reply:
Probably the minor bump - but wanted to entries in the changelog.